src/main/java/gov/nih/nci/evs/api/controller/AuditController.java

```diff
@@ -0,0 +1,100 @@
+package gov.nih.nci.evs.api.controller;
```

Comment:
I don't think we actually want a controller here becuase we don't want to expose the audit trail outside the project (through the public server).  The functions this serve are available internally via ElasticQueryService.  So keep the tests (though maybe rename from AuditControllerTests to just AuditTests - and make the elasticQueryService calls directly)

Reply:
done